fastasyncworldedit/advanced-commands/selection.md

```diff
@@ -0,0 +1,25 @@
+## Introduction
+
+In some cases you want to exclude regions because you don't want to edit them, 
+for example when the unwanted part is inside another wanted part.
+In this example we use the [wesui plugin of kennytv](https://www.spigotmc.org/resources/worldeditsui-visualize-your-selection.60726/) (is optional, but can help to understand each step) to visualize the selections.
```

Comment:
It's enough to only mention the name, but it should be `WorldEditSUI`.

Reply:
Can I leave the link in there? There are 2 plugins that are similar but that's more maintained/active and it is from a known paper developer.

Reply:
Yes, the link is fine.